### PR TITLE
Streamline SSH attack recipe #86

### PR DESCRIPTION
**Pull request by ybyzek**

### CODE REVIEW

File: docs/cybersecurity/SSH-attack/manual.sql [hunk 1]
```diff
@@ -1,9 +1,9 @@
...
 INSERT INTO syslog (ts, host, facility, message, remote_address) VALUES ('2021-08-08T02:11:13.333Z', 'asgard.example.com', 4, 'Invalid user csmith', '192.168.10.83');
-INSERT INTO syslog (ts, host, facility, message, remote_address) VALUES ('2021-08-08T02:11:14.333Z', 'asgard.example.com', 4, 'Invalid user dsmith', '192.168.10.83');
-INSERT INTO syslog (ts, host, facility, message, remote_address) VALUES ('2021-08-08T02:11:15.333Z', 'asgard.example.com', 4, 'Invalid user esmith', '192.168.10.83');
-INSERT INTO syslog (ts, host, facility, message, remote_address) VALUES ('2021-08-08T02:11:16.333Z', 'asgard.example.com', 4, 'Invalid user fsmith', '192.168.10.83');
+INSERT INTO syslog (ts, host, facility, message, remote_address) VALUES ('2021-08-08T02:11:14.333Z', 'asgard.example.com', 4, 'Invalid user dsmith', '192.168.10.84');
+INSERT INTO syslog (ts, host, facility, message, remote_address) VALUES ('2021-08-08T02:11:15.333Z', 'asgard.example.com', 4, 'Invalid user esmith', '192.168.10.85');
+INSERT INTO syslog (ts, host, facility, message, remote_address) VALUES ('2021-08-08T02:11:16.333Z', 'asgard.example.com', 4, 'Invalid user fsmith', '192.168.10.86');
```
Comment (mjsax) [inline docs/cybersecurity/SSH-attack/manual.sql:8]: Why those changes?

Comment (ybyzek) [reply docs/cybersecurity/SSH-attack/manual.sql:8]: For variability and more realistic src IP address spoofing, but not necessary.  In fact, maybe not good if it _should_ be consistent with host being the same.  I can revert

Comment (mjsax) [reply docs/cybersecurity/SSH-attack/manual.sql:8]: I don't now :) Was just wondering.

If @rspurgeon did the original recipe it should be his call.

Comment (rspurgeon) [reply docs/cybersecurity/SSH-attack/manual.sql:8]: My thought was an attacker was trying different user names from the same source IP... but I'm indifferent here.

Comment (ybyzek) [reply docs/cybersecurity/SSH-attack/manual.sql:8]: I'll revert

**Pull request merged**